android/res/layout/routing_bottom_panel_transit.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Replaced `RelativeLayout` with `ConstraintLayout` because is supports max height constraint.